examples/immersive_paris.html

```diff
@@ -162,6 +162,9 @@
             var menuGlobe = new GuiTools('menuDiv');
             menuGlobe.view = view;
 
+            var d = new debug.Debug(view, menuGlobe.gui);
```

Comment:
Change opacity building doesn't work, may be projection material hasn't opacity propertie, no?

Reply:
Yes, thanks, I've updated OrientedImageMaterial and shaders. Now it supports opacity.